packages/runtime/src/useCases/transport/relationships/RelationshipMapper.ts

```diff
@@ -19,39 +19,53 @@ export class RelationshipMapper {
                 publicKey: relationship.peer.publicKey.toBase64(false),
                 realm: relationship.peer.realm
             },
-            changes: relationship.cache.changes.map((c) => this.toRelationshipChangeDTO(c))
+            auditLog: relationship.cache.auditLog.map((entry) => this.toAuditLogEntryDTO(entry)),
+            creationContent: relationship.cache.creationContent?.toJSON()
         };
-    }
-
-    public static toRelationshipDTOList(relationships: Relationship[]): RelationshipDTO[] {
-        return relationships.map((r) => this.toRelationshipDTO(r));
-    }
-
-    private static toRelationshipChangeRequestDTO(change: RelationshipChangeRequest): RelationshipChangeRequestDTO {
         return {
-            createdBy: change.createdBy.toString(),
-            createdByDevice: change.createdByDevice.toString(),
-            createdAt: change.createdAt.toString(),
-            content: change.content?.toJSON()
+            ...dtoWithoutChanges,
+            changes: [
+                {
+                    id: "RCH00000000000000001",
+                    request: {
+                        createdAt: dtoWithoutChanges.auditLog[0].createdAt,
+                        createdBy: dtoWithoutChanges.auditLog[0].createdBy,
+                        createdByDevice: "DVC00000000000000001",
+                        content: dtoWithoutChanges.creationContent
+                    },
+                    status: this.getStatus(dtoWithoutChanges.auditLog),
+                    type: RelationshipChangeType.Creation
+                }
+            ]
         };
     }
```

Comment:
why don't you do this inline?

Reply:
didn't want to have super long dot-chains, but I see they aren't that long anyway